### PR TITLE
1 shake off dust from tbsw

### LINKED ISSUE #1: shake_off_dust_from_tbsw

### PR DESCRIPTION
**Pull request by BenjaminSchwenker**

### CODE REVIEW

File: workspace/scan_runs.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (ybuch) [inline workspace/scan_runs.sh]: I think we should either use the proper bash script with the for loop that we used during tb or delete this

Comment (BenjaminSchwenker) [reply workspace/scan_runs.sh]: agreed. 

i will delete all tjmp2 specific scripts from the default workspace.

**Review by ybuch — DISMISSED** [review]
I don't know if my approval is revoked once you commit the last changes. Lets see

**Comment by BenjaminSchwenker** [comment]
@ybuch : you do have to approve again ;)

**Review by ybuch — APPROVED** [review]

**Pull request merged**